roles/docker/tasks/main.yml

```diff
@@ -17,13 +17,6 @@
 - name: current user is in docker group
   become: yes
   user:
-    name: "{{ ansible_env.USER }}"
+    name: "{{ ansible_user_id }}"
     groups: docker
     append: yes
-
-- name: docker.service is enabled
```

Comment:
need to fix this before I merge this PR

Reply:
Will merge as is as its pretty hard to solve within docker it seems. Enabling this service is not that painful to do anyway and will open an issue to resolve in the future